src/main/java/com/dev/cinema/validation/annotation/EmailValidator.java

```diff
@@ -0,0 +1,19 @@
+package com.dev.cinema.validation.annotation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements
+        ConstraintValidator<EmailValidatorConstraint, String> {
+    @Override
+    public void initialize(EmailValidatorConstraint emailValidatorConstraint) {
+    }
+
+    @Override
+    public boolean isValid(String contactField,
+                           ConstraintValidatorContext cxt) {
+        return contactField != null && contactField.matches("[0-9]+")
+                && (contactField.length() > 8) && (contactField.length() < 14);
```

Comment:
123456789 is valid email? Why email length may be between 8 and 14?

Reply:
+ var naming

Reply:
Changed.